mbuild/packing.py

```diff
@@ -144,6 +147,11 @@ def fill_box(
         Number of compounds to be filled in box.
     box : mb.Box
         Box to be filled by compounds.
+    use_pbc : bool, default=False
+        If True, applies periodic boundary conditions
+        when placing molecules.
+        Changing this to True will automatically change
+        edge to zero.
```

Comment:
I'm not a huge fan of manually overriding arguments with other arguments. I think we should default edge=0, and raise an Error if there is periodicity, but the edge is set to something other than 0.

Reply:
I don't think we should change the default value, especially since use_pbc defaults to False. I added a check for edge values when use_pbc is True.

Reply:
Yeah that's fair, probably better to just raise the error if you try to do one without the other. The error should make it pretty clear what someone needs to do to fix it.